src/cap-description.adoc

```diff
@@ -570,16 +570,14 @@ expanded base is 0 and top is 2^MXLEN^.
 | Reserved | zeros | All reserved fields
 |==============================================================================
 
-If {cheri_default_ext_name} is supported:
-
-* For MXLEN=32, the <<m_bit>> is set to zero (pass:attributes,quotes[{cheri_int_mode_name}]) in the AP field
-* For MXLEN=64, the <<m_bit>> is set to zero (pass:attributes,quotes[{cheri_int_mode_name}]) in a separate M field
+NOTE: <<null-cap>> is not a code capability, and so the CHERI execution mode is not encoded.
```

Comment:
```suggestion
NOTE: For MXLEN=64, the <<m_bit>> is encoded outside of AP and is set to zero for <<null-cap>>.
```

I'd just say we encode M as zero? Alternatively, we could add a `M` row to the table and just add a note there `(not present for MXLEN=32)`: e.g.
```
| M        | zero  | No meaning since non-executable (MXLEN=64 only)
```

Reply:
ok done - will commit shortly